CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-12620: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
   - The permission `WorkflowAdministration` will deprecate the permissions `Policy, VulnerabilityReports`.
 
+- ROX-14400: `BuildDate` is deprecated and will be removed in future releases. It won't be returned by
+`/debug/versions.json` endpoint and not available to use in `roxctl` helm templates.
```

Comment:
Which helm templates?
I thought it's only exposed from `roxctl version --json`.

Reply:
If user use
`roxctl helm output central-services --output-dir /tmp/helm --debug-path <path to custom templates> --debug`
and in template uses `[< .Versions.BuildDate >]` then it will not work

Reply:
> in template uses

I believe we control templates and don't let users modify templates (although they can provide different variables), especially at "`.htpl`" level (`[<` `>]` tags). Therefore, `.Versions.BuildDate` becomes unavailable for us, ACS developers.
Given that we don't seem to use `.Versions.BuildDate`, it does not deserve to be mentioned.

Moreover, the changelog is the communication tool with ACS/StackRox **users**, not developers. We put things that are significant for users to be aware of. Then, the corresponding part of the changelog is converted to product Release Notes by the documentation team. That's how users get to know it.